### DISCUSSION TITLE
Is monika support for monitoring TLS Certificate for SMTP?

### DISCUSSION THREAD

**Discussion by mrofisr** [hyperjumptech/monika#1238] [Ideas]
Hi Folks! I have some case to monitoring the SMTP ( Uptime and TLS Certificate ) does Monika support this?

Thanks
Abdur Rofi

**Comment by sapiderman** [comment]
@mrofisr,

Absolutely! How you can check TLS health here: https://monika.hyperjump.tech/guides/tls-checkers
To check the uptime of an SMTP service, if you have a /health endpoint, would be a good place to start. Another possible API maybe the /login or new mail checking. 

Let us know if you have any difficulties!

**Comment by mrofisr** [comment]
In my SMTP service, I don't have a health check endpoint; it's just a domain and an SMTP port. Currently, I use the OpenSSL CLI to check the expiration date of my TLS certificate in my SMTP.

`openssl s_client -starttls smtp -connect domain.com:587` and `openssl s_client -connect domain.com:465`

From the documentation you provided, it uses `https`. Is it possible to use `tcp:// `with the SMTP port like this: `tcp://domain.com:587`?

Thanks before
